### PR TITLE
Multi company refactor

### PR DESCRIPTION
**Pull request by Samox**

### CODE REVIEW

File: src/redux/reducer.js [hunk 1]
```diff
@@ -77,12 +79,10 @@ export default function reducer(state = initialState, action = {}) {
         errorMessage: action.error,
       };
     case types.COMPANY_SELECTED:
-      const ids = gapi.selectCompany(action.company)
```
Comment (BrunoGodefroy) [inline src/redux/reducer.js:80]: You need to remove scriptId and clientId from the initial state

File: src/components/ui/ListForms.jsx [hunk 2]
```diff
@@ -6,41 +6,31 @@ import { companies } from '../../redux/actions';
 
 class ListForms extends PureComponent {
   render() {
+    const yesOfCourse = this.props.questions.reduce((yes ,question) => {
+      if (question.questionSlug === 'recommendation') {
+        yes = question.answers[0]
+      }
+      return yes
+    }, 'croute')
```
Comment (BrunoGodefroy) [inline src/components/ui/ListForms.jsx]: I think you can replace that by:

```
const yesOfCourse = this.props.questions.find(question => question.questionSlug === 'recommendation').answers[0]
```

File: src/components/ui/ListForms.jsx [hunk 3]
```diff
@@ -6,41 +6,31 @@ import { companies } from '../../redux/actions';
...
-            if (parseInt(form.speed[0]) + parseInt(form.colaboration[0]) == 10){
-              label = <Label color="green" className="pinned">WOW!</Label>;
-            } else {
-              label = <Label color="olive" className="pinned">Success</Label>;
-            }
-          }
-          else {
-            label = <Label color="red" className="pinned">Red bucket</Label>;
-          }
+      if (parseInt(form.speed[0]) + parseInt(form.colaboration[0]) >= 8 && form.recommendation === yesOfCourse) {
```
Comment (BrunoGodefroy) [inline src/components/ui/ListForms.jsx:16]: I think the next step would be to compute that in the backend and add a new field to the form: `successType` that can be either: `wow`, `success` or `red-bucket` 😉

File: src/components/ui/ProjectForm.jsx [hunk 4]
```diff
@@ -15,146 +15,37 @@ class ProjectForm extends PureComponent {
...
+      { this.props.questions.map( question => {
+          switch(question.type) {
+            case 'MULTIPLE_CHOICE':
+              return (
+                <Segment vertical>
+                  <Header as="h3">{ question.label }</Header>
+                  { question.answers.map( answer => {
+                    return (<Form.Radio label={ answer } checked={ this.props.form[question.questionSlug] ===answer } />)
+                  } ) }
+                </Segment>)
```
Comment (BrunoGodefroy) [inline src/components/ui/ProjectForm.jsx]: ❤️

**Review by BrunoGodefroy — CHANGES_REQUESTED** [review]

File: src/components/ui/ListForms.jsx (hunk 3 above)
Comment (Samox) [reply src/components/ui/ListForms.jsx:16]: Aight

**Review by BrunoGodefroy — APPROVED** [review]

**Pull request merged**